File: App/app/src/main/java/com/cinema/ticket_booking/data/model/request/UpdateProfileRequest.java
package com.cinema.ticket_booking.data.model.request;

public class UpdateProfileRequest {
    private String fullName, phone, avatarUrl;

    public UpdateProfileRequest(String fullName, String phone, String avatarUrl) {
        this.fullName = fullName;
        this.phone = phone;
        this.avatarUrl = avatarUrl;
    }
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/request/UpdateProfileRequest.kt
package com.cinema.ticket_booking.data.model.request

import com.google.gson.annotations.SerializedName

data class UpdateProfileRequest @JvmOverloads constructor(
    @SerializedName("fullName")
    var fullName: String? = null,
    @SerializedName("phone")
    var phone: String? = null,
    @SerializedName("avatarUrl")
    var avatarUrl: String? = null
)
